### PR TITLE
Blue green fixes

### PR DESCRIPTION
**Pull request by mmarcotte**
Working with Flexion to get blue/green up and running in the Court environments. This PR aims to add some clarifications to the the documentation and ensure environment variables are set before attempting deploys that rely upon them. Mainly, it's fixing an issue where a script tries to ascertain the name of the source table for a migration.

### CODE REVIEW

File: docs/BLUE_GREEN_MIGRATION.md [hunk 1]
```diff
@@ -1,45 +1,86 @@
...
 Error: Error creating IAM Role migration_role_<ENV>: EntityAlreadyExists: Role with name 		migration_role_<ENV> already exists.
 	status code: 409, request id: ***********
 
   on migration.tf line 1, in resource "aws_iam_role" "migration_role":
    1: resource "aws_iam_role" "migration_role" {
-   ```
-Delete the role in the AWS IAM console and rerun `npm run deploy:environment-specific <ENV>`.
+```
 
+AWS IAM console and rerun `npm run deploy:environment-specific <ENV>`.
```
Comment (julialeague) [inline docs/BLUE_GREEN_MIGRATION.md]: Did you intend to omit `Delete the role` from the beginning of this line?

File: docs/BLUE_GREEN_MIGRATION.md [hunk 2]
```diff
@@ -1,45 +1,86 @@
...
-If the websockets endpoint returns a 403 Unauthorized error (can be seen during trial session smoke tests), redeploy the websocket APIs for the environment in the AWS console under API Gateway (both east and west).
\ No newline at end of file
+
+If the websockets endpoint returns a 403 Unauthorized error (can be seen during trial session smoke tests), redeploy the websocket APIs for the environment in the AWS console under API Gateway (both east and west).
+
+#### Certificates
+
+```Error: Error creating API Gateway Domain Name: BadRequestException: The provided certificate does not exist.
+
+  on ../api/api-public.tf line 106, in resource "aws_api_gateway_domain_name" "api_public_custom":
```
Comment (julialeague) [inline docs/BLUE_GREEN_MIGRATION.md]: This looks to be an unfinished section

**Review by julialeague — APPROVED** [review]
Outside of my two comments, +1

File: docs/BLUE_GREEN_MIGRATION.md (hunk 1 above)
Comment (mmarcotte) [reply docs/BLUE_GREEN_MIGRATION.md]: oops, thanks!

File: docs/BLUE_GREEN_MIGRATION.md (hunk 2 above)
Comment (mmarcotte) [reply docs/BLUE_GREEN_MIGRATION.md]: You're right. It was more importing of existing resources, I think. Anyhow. Yeah, this is a work in progress as I'm going to continue to edit this doc as I get it going in all of the environments. Removed for now.

**Pull request merged**